db/db_iter.cc

```diff
@@ -1117,6 +1110,24 @@ bool DBIter::IsVisible(SequenceNumber sequence, const Slice& ts) {
   }
 }
 
+bool DBIter::IsVisible(const Slice& ts) {
+  return (timestamp_ub_ == nullptr ||
+          user_comparator_.CompareTimestamp(ts, *timestamp_ub_) <= 0) &&
+         (timestamp_lb_ == nullptr ||
+          user_comparator_.CompareTimestamp(ts, *timestamp_lb_) >= 0);
+}
+
+bool DBIter::IsVisible(SequenceNumber sequence, const Slice& ts,
```

Comment:
I think we can just keep `IsVisible(SequenceNumber sequence, const Slice& ts, bool* more_recent)`? Three overloaded version of `IsVisible` seem a little confusing.

Reply:
ok